tools/gyp/PRESUBMIT.py

```diff
@@ -124,3 +123,16 @@ def CheckChangeOnCommit(input_api, output_api):
   finally:
     sys.path = old_sys_path
   return report
+
+
+TRYBOTS = [
+    'linux_try',
+    'mac_try',
+    'win_try',
+]
+
+
+def GetPreferredTryMasters(_, change):
+  return {
+      'client.gyp': { t: set(['defaulttests']) for t in TRYBOTS },
+  }
```

Comment:
What does all of this do, just curious as it's new?

Reply:
Done four years ago in nodejs/node-gyp#781